challenges/objects-arrays.js

```diff
@@ -46,20 +70,31 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
+for (let i=0; i < graduates.length; i++) {
+  universities.push(graduates[i].university);
+}
+universities.sort();
 console.log(universities)
 
 /* Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
-
 The resulting contact information should have a space between the first name and the email information like this: 
 Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+for (let i=0; i < graduates.length, i++) {
+  contactInfo.push(graduates[i].first_name + ' ' + graduates[i].email) 
+}
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+for (let i=0; i < graduates.length, i++) {
```

Comment:
Yet another typo similar to the last one `,` instead of `;` in for loop which results in error.